src/applications/gqrx/mainwindow.cpp

```diff
@@ -1597,11 +1597,14 @@ void MainWindow::startIqPlayback(const QString& filename, float samprate)
     }
 
     storeSession();
+    backupFreq = ui->freqCtrl->getFrequency();
+    backupOffset = (qint64) rx->get_filter_offset();
```

Comment:
Why is it necessary to add these variables? Doesn't `storeSession()` already save them in the `input/frequency` and `receiver/offset` settings?

Reply:
Using the "Device dialog" or the "Save settings" button may overwrite settings during IQ playback as both are calling `storeSession()`. But using any of these buttons breaks playback too...
OK. Ill switch to reading center/offset from stored settings and add disabling "Save settings" button during playback to https://github.com/gqrx-sdr/gqrx/pull/1011.

Reply:
The suggestion to use `storeSession()` was just for consistency with the current design. I agree that the current design is not ideal, so maybe we should think about changing it (in a future PR, perhaps).